cms/templates/js/asset-library.underscore

```diff
@@ -13,10 +14,46 @@
         <thead>
         <tr>
             <th class="thumb-col"><%= gettext("Preview") %></th>
-            <th class="name-col sortable-column"><span class="column-sort-link" id="js-asset-name-col"><%= gettext("Name") %></span></th>
-            <th class="date-col sortable-column"><span class="column-sort-link" id="js-asset-date-col"><%= gettext("Date Added") %></span></th>
-            <th class="embed-col"><%= gettext("Embed URL") %></th>
-            <th class="embed-col"><%= gettext("External URL") %></th>
+            <th class="name-col sortable-column">
+                <span class="column-sort-link" id="js-asset-name-col" role="button" tabindex="0">
+                    <%= gettext("Name") %>
+                    <span class="sr"><%= gettext("- Sortable") %></span>
+                </span>
+            </th>
+            <th class="type-col filterable-column nav-dd">
+                <div id="js-asset-type-col" class="nav-item" role="button" tabindex="0">
+                    <span class="title">
+                        <span class="type-filter" data-alllabel='<%= gettext("Type") %>'><%= gettext("Type") %></span>
+                        <span class="label-prefix sr">Filter</span>
+                        <span class="filter-link"></span>
+                        <i class="icon-caret-down ui-toggle-dd" aria-hidden="true"></i>
```

Comment:
All font-awesome classes have to be converted to the new naming convention (http://fortawesome.github.io/Font-Awesome/icon/caret-down/).